src/components/atoms/OcDrop/OcDrop.vue

```diff
@@ -208,6 +208,27 @@ export default {
     // note: needed so that the box shadow from `oc-box-shadow-medium` doesn't get suppressed
     padding: 8px;
   }
+  li.action-menu-item-hover {
```

Comment:
I have one last issue. Could you please use `oc-menu-item-hover` as class name? Otherwise we have no chance of knowing in the consuming code location (in web) that this class comes from the ODS. That's the reason why all design system classes are prefixed with `oc-`.